src/tabs/ux-tab-theme.css

```diff
@@ -0,0 +1,34 @@
+styles.tab {
+  display: inline-block;
+  text-transform: uppercase;
+  cursor: pointer;
+  opacity: 0.7;
+  padding: 14px 12px 20px 12px;
+  text-align: center;
+  color: ${foreground || $design.accentForeground};
+  background-color: ${background || $design.accent};
+}
+
+styles.tab.disabled {
+  cursor: not-allowed;
+}
+
+styles.tab:hover:not(.disabled) {
+  opacity: 1;
+}
+
+styles.tab.selected:not(.disabled) {
+  opacity: 1;
+  padding-bottom: 18px;
+  border-bottom: ${foreground || $design.accentForeground} solid 2px;
```

Comment:
The indicator should be `$design.accent` if we are going to base this on the accent color. `$design.accentForeground` is not always `#fff`. In addition, I would also like to see `foreground` changed to a new property specific to the indicator that can be set in the .ts theme.

> Indicator
>
> - Height: 2dp
> - Color: #fff or accent color

[Material Guidelines - Tabs](https://material.io/guidelines/components/tabs.html#tabs-specs)